### PR TITLE
Milestone 1

### PR DESCRIPTION
**Pull request by dami1080**
In this project, I have set up a JavaScript project for the Leaderboard list app, using webpack and ES6 features. I developed the first working version of the app following the given wireframe.
Based on the project requirements, I have done the following:

- Created an npm project with webpack.
- Wrote plain HTML markup with minimum styling (just to make the layout).
- Used the wireframe below as a guide:
- https://raw.githubusercontent.com/microverseinc/curriculum-javascript/main/leaderboard/images/leaderboard_wireframe.png?token=AGMOBB2DHPDSQOPSQE3RWEDBG4636

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+<script defer src="main.js"></script></head>
+<body>
+    <div class="header">
+        <h1>Leaderboard</h1>
+    </div>
+
+    <div class="score-container">
+    
+    </div>
+
```
Comment (vichuge) [inline dist/index.html]: - [ ] Please, make sure that index.html should only have a root element. 🐶

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your PR messages are descriptive ✔️

## Aspects to improve ♻️

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: dist/index.html (hunk 1 above)
Comment (henatan99) [reply dist/index.html]: This issue is well addressed. Great work! :clap: :clap:

**Review by henatan99 — APPROVED** [review]
Hi @dami1080 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**